js/src/embed.js

```diff
@@ -0,0 +1,118 @@
+// Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import 'babel-polyfill';
+import 'whatwg-fetch';
+
+import es6Promise from 'es6-promise';
+es6Promise.polyfill();
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
+
+import injectTapEventPlugin from 'react-tap-event-plugin';
+
+import SecureApi from './secureApi';
+import ContractInstances from '~/contracts';
+
+import { initStore } from './redux';
+import ContextProvider from '~/ui/ContextProvider';
+import muiTheme from '~/ui/Theme';
+
+import { patchApi } from '~/util/tx';
+import { setApi } from '~/redux/providers/apiActions';
+
+import './environment';
+
+import '../assets/fonts/Roboto/font.css';
+import '../assets/fonts/RobotoMono/font.css';
+
+injectTapEventPlugin();
+
+import ParityBar from './views/ParityBar';
```

Comment:
Mixture of `~` (i.e. ui as used above) and `./` - doesn't make much difference here since it sits at the root, just noting it that generally we prefer `~` and if things are moved, it won't have any impact.